channels/serializers.py

```diff
@@ -659,6 +667,14 @@ def validate_moderator_name(self, value):
             raise ValidationError("moderator name is not a valid user")
         return {'moderator_name': value}
 
+    def validate_email(self, value):
```

Comment:
Maybe move the bulk of this code into a separate function called by both `Contributor.validate_email` and `Moderator.validate_email`, to avoid repetition?

```
    def validate_email(self, value):
        """Validates the contributor email"""
        return is_valid_email(value)
```

```
def is_valid_email(value):
    if not isinstance(value, str):
        raise ValidationError("email must be a string")
    if not User.objects.filter(email__iexact=value).exists():
        raise ValidationError("email does not exist")
    return {'email': value}
```